OsiguSDK.Providers/Models/V1/Diagnosis.cs

```diff
@@ -9,13 +9,13 @@ public class Diagnosis
         /// Dianosis ID
         /// </summary>
         [JsonProperty(PropertyName = "id")]
-        public string ProductId { get; set; }
+        public decimal Id { get; set; }
```

Comment:
it this supposed to be decimal or just int?

Reply:
I would avoid `int`, we are supposed to use long values as IDs, from the API side.

Reply:
Yep you're right +1